OpenCore-Catalina/config-pt.plist

```diff
@@ -0,0 +1,886 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@kholia Added a new configuration file instead of a new directory altogether, also generated the QEMU disk image with the generator script and set it as default in the passthrough script.

Just after I rebased the repo to your latest changes, it again had merge conflicts which I resolved in-browser.

Reply:
Thanks. Please squash your commits into one single commit. And we can restart the review process.

Reply:
We are almost all set :)

Reply:
I've squashed all the commits into one commit, you can see the breakdown of the constituent commits at https://github.com/kittywhiskers/OSX-KVM/tree/upstream-old